### ISSUE TITLE
Indentation rule behaviour

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#225]
One thing I've noticed as since switch from SCSSLint is that the indentation rule behaves slightly different in SassLint. While I know there are some issues with maps, it also applies the rule to comments.

While I don't think this behaviour is any less correct, I think it would be great if we could have a option to disregard incorrectly indented comments.

An example of where this could happen:

<img width="759" alt="screen shot 2015-09-26 at 23 17 26" src="https://cloud.githubusercontent.com/assets/2854338/10120131/c78db2ee-64a4-11e5-88a3-42ceb6ca33de.png">

SassLint - develop

**Comment by DanPurdy** [comment]
Yeah i think the option to ignore comments would be a welcome addition. Im leaning more towards them being ignored by default with an option to include them.

**Comment by Snugug** [comment]
I'd prefer them not ignored by default as it's an exception to the rule

**Comment by DanPurdy** [comment]
Yeah makes sense, lets go with that.
